webanno-ui-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/project/ProjectUsersPanel.java

```diff
@@ -354,6 +358,20 @@ public void onSubmit()
         public UserDetailForm(String id)
         {
             super(id, new CompoundPropertyModel<SelectionModel>(new SelectionModel()));
+            TextField<String> filterText = new TextField<String>("userfilter");
+            
+            add(filterText.setOutputMarkupPlaceholderTag(true));
+            add(new Button("filterbutton")
+            {
+            	private static final long serialVersionUID = -7523594952670514192L;
+            	
+            	@Override
+				public void onSubmit()
+                {
```

Comment:
Please use this Eclipse code style file: http://dkpro.github.io/files/DKProCoreStyle_20120326.xml

Please do not reformat code outside the lines that you are changing for functional reasons, even if there are lines that do not conform to the code style file.